python/ngraph_bridge/__init__.in.py

```diff
@@ -223,5 +218,4 @@ def get_disabled_ops():
     "nGraph version used for this build: " + str(ngraph_bridge_lib.ngraph_lib_version()) + "\n" + \
```

Comment:
would this change?

Reply:
Change to what? This is just printing out the version of nGraph in use.

Reply:
nGraph version won't change anymore, right?
So we don't need to display it.

Reply:
It's not under our control, but I agree that it isn't particularly useful to show.

Reply:
when building with OV I have noticed that the version is 0.0.0+<some-commit-id>
eg: nGraph version used for this build: b'0.0.0+a5f5384' which is why I assumed it wouldn't change

Reply:
It identifies the git commit which changes with every version.

But if you notice carefully, our integration (build scripts, source, etc.) are still nGraph-specific; this PR only changes where the nGraph we build against comes from. Should we revisit the suggested change above as part of the OV rebranding?

Reply:
Sure, we can do that.